docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -80,7 +80,7 @@ teku validator generate --deposit-amount-gwei=<GWEI>
 teku validator generate --deposit-amount-gwei=32000000000
 ```
 
-Amount to deposit in the Ethereum 1.0 deposit contract.
+Amount to deposit in the Ethereum 1.0 deposit contract. Defaults to 32 ETH.
```

Comment:
Actually defaults to the minimum ETH required to activate a validator on the specified network.  That's 32ETH for most networks, but only 3.2 when using `--network minimal` config or any network based on it.

Reply:
Done